DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Services/ScalingServiceToHost.cs

```diff
@@ -0,0 +1,37 @@
+using Autodesk.Revit.DB;
+using Speckle.Converters.Common;
+
+namespace Speckle.Converters.RevitShared.Services;
+
+public sealed class ScalingServiceToHost
+{
+  public double ScaleToNative(double value, string units)
+  {
+    if (string.IsNullOrEmpty(units))
+    {
+      return value;
+    }
+
+    return UnitUtils.ConvertToInternalUnits(value, UnitsToNative(units));
+  }
+
+  public ForgeTypeId UnitsToNative(string units)
+  {
+    var u = Core.Kits.Units.GetUnitsFromString(units);
```

Comment:
I thought we had other ways of doing this conversion? I think in other places we're using a dictionary which feels a bit more terse?

Reply:
I was referring to this class? does the achieve the same thing?